SMITHY_GO_CODEGEN_VERSION

```diff
@@ -1 +1 @@
-695755c8eb501de6899f71cd19e8249daff8f8b9
+6f12c095f5277d7e682217bcfd50bab607b193ab
```

Comment:
I double checked this commit id matches on Smithy-Go https://github.com/aws/smithy-go/commit/6f12c095f5277d7e682217bcfd50bab607b193ab

Reply:
Yea, the main branch commit after smithy go merge